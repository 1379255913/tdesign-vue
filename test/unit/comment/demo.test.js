import { mount } from '@vue/test-utils';
import demo from '@/examples/comment/demos/base.vue';

// unit test for component in examples.
describe('Comment', () => {
  it('base demo works fine', () => {
    const wrapper = mount(demo);
    expect(wrapper.element).toMatchSnapshot();
  });
});
