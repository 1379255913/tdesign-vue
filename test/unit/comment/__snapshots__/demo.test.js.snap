// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment base demo works fine 1`] = `
<div
  class="t-comment"
>
  <div
    class="t-comment__inner"
  >
    <div
      class="t-comment__avatar"
    >
      <img
        alt=""
        class="t-comment__avatar-image"
        src="https://tdesign.gtimg.com/list-icon.png"
      />
    </div>
    <div
      class="t-comment__content"
    >
      <div
        class="t-comment__author"
      >
        <span
          class="t-comment__name"
        >
          评论作者名
        </span>
        <span
          class="t-comment__time"
        >
          今天16:38
        </span>
      </div>
      <div
        class="t-comment__detail"
      >
        评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容评论内容。
      </div>
    </div>
  </div>
</div>
`;
